state/backups/metadata.go

```diff
@@ -68,6 +83,12 @@ type Metadata struct {
 	// Notes is an optional user-supplied annotation.
 	Notes string
 
+	// FormatVersion stores format version of these metadata.
+	FormatVersion int64
+
+	// Controller contains metadata about the controller of the backup.
```

Comment:
Maybe "Controller contains metadata about the controller the backup was taken from."?